File: CMOWebApi.Models/AdminModel/MasterModel/helpDocumentModel.cs
using System;

namespace CMOWebApi.Models.AdminModel.MasterModel
{
    public class HelpDocumentModel
    {
        public long Id { get; set; }

        public long TypeCode { get; set; }

        public string Url { get; set; }
		public string BlankDocUrl { get; set; }
		public bool IsImageChange { get; set; }

		public bool IsBlankDocChanges { get; set; }

		public Nullable<bool> IsActive { get; set; } = true;

        public Nullable<bool> IsDelete { get; set; } = false;

        public Nullable<System.DateTime> CreatedDate { get; set; }

        public Nullable<int> CreatedBy { get; set; }

        public Nullable<System.DateTime> ModifiedDate { get; set; }

        public Nullable<int> ModifiedBy { get; set; }
        public string DocumentName { get; set; }
        public Nullable<int> DisplayOrder { get; set; }
    }

    public class HelpDocumentViewModel
    {
        public long Id { get; set; }

        public string TypeName { get; set; }

        public string Url { get; set; }
		public string BlankDocUrl { get; set; }

		public string SSOID { get; set; }
        public Nullable<System.DateTime> ModifiedDate { get; set; }
        public string modifiedByName { get; set; }
        public string DocumentName { get; set; }
        public Nullable<int> DisplayOrder { get; set; }
    }
}

File: ANGULAR/ANGULAR/src/app/Shared/Model/Master/helpdocument.model.ts
export class HelpDocumentModel {
  Id: number;
  TypeCode: number | string;
  Url: string;
  IsImageChange: boolean;
  BlankDocUrl: string;
  IsBlankDocChanges: boolean;
  DocumentName: string;
  DisplayOrder: number;
  constructor() {
    this.Id = 0;
    // this.TypeCode = 0;
    this.Url = '';
    this.IsImageChange = false;
    this.BlankDocUrl = '';
    this.IsBlankDocChanges = false;
    this.DocumentName = '';
  }
}


export class HelpDocumentViewModel {
  Id?: number;
  TypeName: string;
  Url: string;
  BlankDocUrl: string;
  ModifiedDate: Date;
  modifiedByName: string;
  DocumentName: string;
  DisplayOrder: string;
}
